tests/util.py

```diff
@@ -61,7 +61,7 @@ def is_alive(self):
 
     def stop(self):
         if self.is_alive():
-            self.process.terminate()
+            self.process.kill()
```

Comment:
hupper now catches SIGTERM so instead we want to just SIGKILL in the test suite once we're done with the test